modules/format_strawberryfield_views/src/Plugin/views/filter/AdvancedSearchApiFulltext.php

```diff
@@ -578,18 +578,13 @@ public function buildExposedForm(&$form, FormStateInterface $form_state) {
         '#default_value' => $form_state->getValue($searched_fields_identifier),
       ];
 
-      if (empty($form[$this->options['id'] . '_wrapper'])) {
-        $form[$this->options['id'] . '_wrapper'] =
-          [
-            "#type" => "fieldset",
-            "#title" => t('Advanced Fulltext search'),
-            $this->options['id'] => $form[$this->options['id']],
-           ];
+      if (empty($form[$this->options['id'] . '_wrapper']) && !empty($form[$this->options['id']])) {
```

Comment:
Almost. I mean it does work, but if you check the method you don't need too check if the wrapper is already there or not, the  $this->buildValueWrapper() does it already for you.

Reply:
I put in the check because in the next line we make this assignment:
```
$form[$this->options['id'] . '_wrapper'][$this->options['id']] = $form[$this->options['id']];
```
If the form has already had the wrapper provided (e.g. if you elect to have exposed the operator), then `$form[$this->options['id']]` will be empty and would cause an error.

Reply:
Ok, I realize the mistake in my argument. Can we always assume the absence of the wrapper element guarantees that `$form[$this->options['id']]` exists? `buildValueWrapper` does not, by itself, guarantee it.

Reply:
Ok - removed.